firefox-ios/firefox-ios-tests/Tests/ClientTests/MainMenu/MainMenuConfigurationUtilityTests.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Common
+import Shared
+
+@testable import Client
+
+final class MainMenuConfigurationUtilityTests: XCTestCase {
+    var configUtility: MainMenuConfigurationUtility!
+    let windowUUID: WindowUUID = .XCTestDefaultUUID
+
+    override func setUp() {
+        super.setUp()
+        configUtility = MainMenuConfigurationUtility()
```

Comment:
If we are setting this up, let's also add a teardown method where we set 
`configUtility = nil`